sql-cli/CONTRIBUTING.md

```diff
@@ -52,7 +52,6 @@ Internally, what happens is that we build a command that we pass on to a docker
 Additionally, the `flow` command requires some more functionality like mounting files from the host to the container to be able to use the created sql-cli project outside of the container after the command exits. Furthermore, we resolve directory paths before mounting so that we have the exact same directory structure outside and inside the container.
 _You can find the code for that in `sql` directory._
 
-_You can find the user documentation of the "flow" command [here](https://docs.astronomer.io/astro/cli/sql-cli)._
```

Comment:
I guess we no longer need this here considering https://github.com/astronomer/astro-sdk/pull/1917